x/slashrefund/keeper/deposit_test.go

```diff
@@ -66,14 +39,87 @@ func TestDepositRemove(t *testing.T) {
 
 func TestDepositGetAll(t *testing.T) {
 	keeper, ctx := testslashrefund.NewTestKeeper(t)
-	items := createNDeposit(keeper, ctx, 10)
+	items := testslashrefund.CreateNDeposit(keeper, ctx, 10)
 	require.ElementsMatch(t, items, keeper.GetAllDeposit(ctx))
 }
 
 func TestGetValidatorDeposits(t *testing.T) {
 	keeper, ctx := testslashrefund.NewTestKeeper(t)
-	items0, valAddr0 := createNDepositForValidator(keeper, ctx, 10)
-	items1, valAddr1 := createNDepositForValidator(keeper, ctx, 10)
+	items0, valAddr0 := testslashrefund.CreateNDepositForValidator(keeper, ctx, 10)
+	items1, valAddr1 := testslashrefund.CreateNDepositForValidator(keeper, ctx, 10)
 	require.ElementsMatch(t, items0, keeper.GetValidatorDeposits(ctx, valAddr0))
 	require.ElementsMatch(t, items1, keeper.GetValidatorDeposits(ctx, valAddr1))
 }
+
+// -------------------------------------------------------------------------------------------------
+// Test deposit pool
+// -------------------------------------------------------------------------------------------------
+
+func TestPoolGetDeposit(t *testing.T) {
```

Comment:
Should be renamed in TestGetDepositPool since the tested function is GetDepositPool().

Reply:
right